instances/treasury-devdao.near/widget/lib/common.jsx

```diff
@@ -39,14 +44,14 @@ function getApproversAndThreshold(treasuryDaoID, kind) {
         denominator += value;
       }
     });
-  } else {
-    numerator = 1;
-    denominator = 2;
   }
+  const approverAccounts = Array.from(new Set(approversGroup));
 
   return {
-    approverAccounts: Array.from(new Set(approversGroup)),
-    threshold: numerator / denominator,
+    approverAccounts,
+    requiredVotes:
+      requiredVotes ||
+      Math.floor((numerator / denominator) * approverAccounts.length) + 1,
```

Comment:
I'd recommend you to consider implementing the exact logic from the contract here and use the same terminology. https://github.com/near-daos/sputnik-dao-contract/blob/3d568f9517a8c7a6510786d978bb25b180501841/sputnikdao2/src/policy.rs#L407-L425

When WeightVote is not involved, the requiredVotes is:

```
max(
  min(number_of_voters, quorum),
  min(
    floor(threshold.ratio * number_of_voters) + 1,
    number_of_voters
  )
)
```

I think currently, quorum rules are not taken into account at all.